.env.sample

```diff
@@ -7,6 +7,7 @@ GOOGLE_TAG_MANAGER_KEY=''
 S3_ACCESS_KEY=''
 S3_ACCESS_KEY_ID=''
 SHOW_SALE_LISTINGS=true
+SHOW_PRE_LOTTERY_INFO=false
```

Comment:
Does this also need to be added to the `app.json` file? I'm not really sure what `app.json` is used for.

Reply:
Not sure what is it for, but good catch, thanks.